lib/js/components/Headers/SectionHeader/SectionHeader.vue

```diff
@@ -0,0 +1,343 @@
+		:class="{
+			'-expandable': expandable,
+			'-horizontal': mobileLayout === SECTION_HEADER_MOBILE_LAYOUTS.HORIZONTAL,
+			[sizeClass]: true,
+		}"
+	>
+		<div class="sectionHeader__wrapper">
+			<div class="sectionHeader__main">
+				<div class="sectionHeader__titleWrapper" @click="onTitleWrapperClicked">
+					<div v-if="iconLeft" class="sectionHeader__iconWrapper sectionHeader__iconLeft">
+						<ds-icon :icon="iconLeft" :size="iconSize" />
+					</div>
+					<span class="sectionHeader__title">{{ title }}</span>
+					<div
+						v-if="iconRight"
+						class="sectionHeader__iconWrapper sectionHeader__iconRight"
+					>
+						<ds-icon :icon="iconRight" :size="iconSize" />
+					</div>
+					<div
+						v-if="expandable"
+						class="sectionHeader__iconWrapper sectionHeader__chevron"
+					>
+						<ds-icon
+							:icon="ICONS.FA_CHEVRON_DOWN"
+							:rotation="chevronRotation"
+							:size="iconSize"
+						/>
+					</div>
+					<div v-if="info" class="sectionHeader__info">
+						<ds-icon-button
+							:icon="ICONS.FA_CIRCLE_QUESTION"
+							:size="ICON_BUTTON_SIZES.X_SMALL"
+							:color="ICON_BUTTON_COLORS.NEUTRAL_WEAK"
+							:touchable="false"
+							@click.prevent.stop="onInfoClicked"
+						/>
+					</div>
+				</div>
+				<div
+					v-if="$slots.default && showSlot"
+					class="sectionHeader__slotHorizontal -hideOnMobile"
+				>
+					<slot />
+				</div>
+			</div>
+
+			<div v-if="supportingText" class="sectionHeader__supportingText"
+				>{{ supportingText }}
+			</div>
+
+			<div
+				v-if="$slots.default && showSlot"
+				class="sectionHeader__slotVertical -showOnMobile"
+			>
+				<slot />
+			</div>
+		</div>
+		<ds-divider v-if="divider" />
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/typography/tokens';
+@import '../../../../styles/settings/media-queries';
+@import '../../../../styles/settings/spacings';
+
+.sectionHeader {
+	$root: &;
+
+	padding: 0;
+
+	&__wrapper {
+		display: flex;
+		flex-direction: column;
+		padding: 0;
+	}
+
+	&__main {
+		align-items: center;
+		display: flex;
+		flex-direction: row;
+		gap: $space-xxs;
+		justify-content: space-between;
+		padding: $space-xxxxs 0;
+	}
+
+	&__supportingText {
+		@include text-s-compact-regular;
+
+		color: $color-neutral-text;
+		gap: $space-xxs;
+		padding: 0 0 $space-xs;
+	}
+
+	&__titleWrapper {
+		align-items: center;
+		display: flex;
+		width: 100%;
+
+		@media #{breakpoint-s()} {
+			width: auto;
+		}
+	}
+
+	&.-expandable &__titleWrapper {
+		cursor: pointer;
+	}
+
+	&.-expandable &__titleWrapper:hover {
+		#{$root}__iconLeft,
+		#{$root}__iconRight,
+		#{$root}__chevron {
+			color: $color-neutral-icon-hovered;
+		}
+
+		#{$root}__title {
+			color: $color-neutral-text-hovered;
+		}
+	}
+
+	&__iconLeft,
+	&__iconRight,
+	&__chevron {
+		color: $color-neutral-icon;
+	}
+
+	&__title {
+		color: $color-neutral-text-strong;
+		padding: $space-xxs 0;
```

Comment:
can this padding be added to `__main` selector's padding?

Reply:
Ah. I see it is to keep space between info as well. So maybe we should use `row-gap` in `__main` as well

Reply:
or add padding from `__main` to this one